dev/reference/tools/loader.md

```diff
@@ -97,6 +97,14 @@ user = "root"
 password = ""
 port = 4000
 
+# sql_mode used when loading data. If not provided or set to "@DownstreamDefault", downstream global sql_mode will be used
```

Comment:
If what is not provided or set to "@DownstreamDefault"? downstream global sql_mode?

Reply:
addressed, PTAL